geps/gep-1742.md

```diff
@@ -1,7 +1,7 @@
 # GEP-1742: Timeouts
 
 * Issue: [#1742](https://github.com/kubernetes-sigs/gateway-api/issues/1742)
-* Status: Provisional
+* Status: Implementable
```

Comment:
@shaneutt, it's been a couple of weeks now, and we've changed it to be Extended support, so I'm suggesting that we can make it Implementable now, so that we can also merge the related interfaces PR (#2013).

See https://github.com/kubernetes-sigs/gateway-api/pull/1997#issuecomment-1578812611

Let me know if you think it's still too soon, and I'll switch it back to Provisional.

Reply:
Ok I think I just missed the comment. I can be convinced that we should move forward with `Implementable` at this point, I just wanna make sure nobody else has any confusion about that.

/cc @robscott @youngnick

Reply:
I think going direct to implementable makes sense here, but you'll also need to make a corresponding change to update the status in the navigation as well:

https://github.com/kubernetes-sigs/gateway-api/blob/8afce4ea7223d85aef23b93e212a115b41d06a72/mkdocs.yml#L82